### DISCUSSION TITLE
Alchemy Signup problem

### DISCUSSION THREAD

**Discussion by vicky4421** [Cyfrin/foundry-full-course-cu#2386] [Q&A]
While signing up to Alchemy.com via google getting error
```
blocked signup
```
i've never created an account before.

**Comment by EngrPips** [comment]
Hello @vicky4421, Please report this to Alchemy. This https://discord.com/invite/alchemyplatform is their official discord

**Reply by vicky4421** [reply]
thank you for quick response, i've reported it, lets see what they've to say.

**Reply by EngrPips** [reply]
You are welcome, Friend.

**Comment by s3bc40** [comment]
Any update on this issue ? I have the same one since I have reached this part of the course :smiling_face_with_tear:

**Reply by EngrPips** [reply]
Do you know you can use other url providers such as `Infura` and `QuickNode`

**Reply by s3bc40** [reply]
I'll do this to continue the course thanks 🙏

I followed what you suggested above : I have made a ticket on the Alchemy discord.

If I have any feedback I'll post it in here 👍

**Reply by EngrPips** [reply]
Awesome, Looking forward to any updates.

**Reply by vicky4421** [reply]
I've tried different email and it worked!

**Reply by EngrPips** [reply]
Interesting, I am glad to hear this.

**Reply by s3bc40** [reply]
From the suppor team of Alchemy : 

> Due to security protocols, signups for Alchemy require a unique, non-temporary email address to sign up. Please try again with a different email address, and we recommend avoiding gibberish or numeric emails.

So if we use an email corresponding to what they expect, it will work. Mine have numbers inside. I went with Infura for the course.

If you reach this github thread and you want to use Alchemy : use an email with only letters inside, no numbers or special chars :pray:

**Reply by EngrPips** [reply]
> From the suppor team of Alchemy :
> 
> > Due to security protocols, signups for Alchemy require a unique, non-temporary email address to sign up. Please try again with a different email address, and we recommend avoiding gibberish or numeric emails.
> 
> So if we use an email corresponding to what they expect, it will work. Mine have numbers inside. I went with Infura for the course.
> 
> If you reach this github thread and you want to use Alchemy : use an email with only letters inside, no numbers or special chars 🙏

Interesting, Thanks for the heads-up buddy.
